README.md

```diff
@@ -83,6 +83,7 @@ Here is a list of all of the supported languages:
 | 🇫🇷🇨🇦 | `Lang::French`    | `fr`      | French      | quarante-deux |
 | 🇧🇪🇨🇩 | `Lang::French_BE` | `fr_BE`   | French (BE) | quarante-deux |
 | 🇨🇭   | `Lang::French_CH` | `fr_CH`   | French (CH) | quarante-deux |
+| 🇪🇸   | `Lang::Spanish`   | `es`      | Spanish     | cuarenta y dos|
```

Comment:
do put Spanish above French! I am thinking of ranking that per total number of speakers, and Spanish has more people speaking it than French

also thinking adding the flag of Mexico after the Spanish flag would make sense

Reply:
I originally put it below french because of alphabetic order. Curiously until now, popularity and alphabetic order matched lol
The mexican flag sounds alright. I just hope they don't have differing numbering (which remembers me, I need to see how to translate currency)